modules/planning/tasks/deciders/path_bounds_decider.h

```diff
@@ -34,6 +34,13 @@
 namespace apollo {
 namespace planning {
 
+// PathBoundPoint contains: (s, l_min, l_max).
+using PathBoundPoint = std::tuple<double, double, double>;
+// PathBoundary contains a vector of PathBoundPoints.
+using PathBoundary = std::vector<PathBoundPoint>;
+// ObstacleEdge contains: (is_start_s, s, l_min, l_max, obstacle_id).
+using ObstacleEdge = std::tuple<int, double, double, double, std::string>;
+
```

Comment:
It is not recommended to put "using xx = yy" in header file. This will be exposed to other files that include this header.

Reply:
fixed. this.